polaris/polaris/templates/base.html

```diff
@@ -44,87 +44,93 @@
       {% endblock %}
     </div>
     <script type="text/javascript">
-"use strict";
-function _instanceof(left, right) { if (right != null && typeof Symbol !== "undefined" && right[Symbol.hasInstance]) { return !!right[Symbol.hasInstance](left); } else { return left instanceof right; } }
+      "use strict";
```

Comment:
Lets figure out a formatter to use for html so we don't get these whitespace changes in PRs.